src/dev/LCD.cpp

```diff
@@ -0,0 +1,95 @@
+#include <EVT/dev/LCD.hpp>
+
+namespace EVT::core::DEV {
+
+LCD::LCD(EVT::core::IO::GPIO& regSelect, EVT::core::IO::GPIO& reset, EVT::core::IO::SPI& spi, unsigned char* bitMap)
+    : regSelect(regSelect), reset(reset), spi(spi) {
+    this->regSelect.writePin(EVT::core::IO::GPIO::State::LOW);
+    this->reset.writePin(EVT::core::IO::GPIO::State::LOW);
+    this->spi = spi;
+}
+
+void LCD::dataWrite(unsigned char data) {
+    this->regSelect.writePin(EVT::core::IO::GPIO::State::HIGH);
+    this->spi.startTransmission(0);
+    this->spi.write(&data, 1);
+    this->spi.endTransmission(0);
+}
+
+void LCD::commWrite(unsigned char data) {
+    this->regSelect.writePin(EVT::core::IO::GPIO::State::LOW);
+    this->spi.startTransmission(0);
+    this->spi.write(&data, 1);
+    this->spi.endTransmission(0);
+}
+
+void LCD::drivePixel(unsigned char page, unsigned char col_up, unsigned char col_low, unsigned char data) {
+    this->commWrite(0x40);          //line to start writing on (0 -> 64) moves set bits with it DO NOT CHANGE
+    this->commWrite(0xB0 + page);   //writes the page address (4 bits, 8 rows selcted by values 0-7 )
+    this->commWrite(0x10 + col_up); //writes the first 4 bits of the column select (out of 8 bits)
+    this->commWrite(0x00 + col_low);//writes the second 4 bits of the column select (out)
+
+    this->dataWrite(data);//writes 8 vertical bits based on value between 0-255 based on bits set ex: 01001100(0x4C) is |WHITE|
+                          //                                                                                                 |BLACK|
+                          //                                                                                                 |WHITE|
+                          //                                                                                                 |WHITE|
+                          //                                                                                                 |BLACK|
+                          //                                                                                                 |BLACK|
+                          //                                                                                                 |WHITE|
+                          //                                                                                                 |WHITE|
+}
+
+void LCD::clearLCD(unsigned char* bitMap) {
```

Comment:
This is fine for now, but I'm willing to bet there's a better way to handle this than manually writing a 0 into every pixel